sdk/storage/azblob/blockblob/models.go

```diff
@@ -72,8 +74,12 @@ type StageBlockOptions struct {
 
 	LeaseAccessConditions *blob.LeaseAccessConditions
 
-	// Specify the transactional crc64 for the body, to be validated by the service.
-	TransactionalContentCRC64 []byte
+	// Let the SDK hash for you (providing the hash type(s) specified voids their relevance in this option).
+	// nil = None default
+	TransactionalValidationOption hashing.StorageTransferValidationOption
```

Comment:
Does this change also need to be made in `uploadFromReaderOptions` and `UploadStreamOptions` types?